### PR TITLE
VACMS-4355 VC facility health service

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4355 

## Testing done
Visual, behat, phpunit

## Screenshots
image

## QA steps
- As an administrator, go to `node/add/vet_center_facility_health_servi`
- Select Boston Vet Center, and choose the Complementary and alternative medicine options.
- Save the form, and visually verify the page and path match the screenshot, above.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE` prior to https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4366 being merged

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/field.field.node.vet_center_facility_health_servi.field_vet_center.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  entity_reference_validators:
+    circular_reference: false
+    duplicate_reference: false
+id: node.vet_center_facility_health_servi.field_vet_center
+field_name: field_vet_center
+entity_type: node
+bundle: vet_center_facility_health_servi
+label: 'Vet Center'
+description: ''
+required: false
```
Comment (kevwalsh) [inline config/sync/field.field.node.vet_center_facility_health_servi.field_vet_center.yml:21]: should be required (i just updated AC, sorry)

File: config/sync/field.field.node.vet_center_facility_health_servi.field_service_name_and_descripti.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  entity_reference_validators:
+    circular_reference: false
+    duplicate_reference: true
+id: node.vet_center_facility_health_servi.field_service_name_and_descripti
+field_name: field_service_name_and_descripti
+entity_type: node
+bundle: vet_center_facility_health_servi
+label: Service
+description: ''
+required: false
```
Comment (kevwalsh) [inline config/sync/field.field.node.vet_center_facility_health_servi.field_service_name_and_descripti.yml:21]: Should also be required

**Review by kevwalsh — CHANGES_REQUESTED** [review]
Create_Vet_Center_Facility_Health_Service___VA_gov_CMS

File: config/sync/field.field.node.vet_center_facility_health_servi.field_vet_center.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/field.field.node.vet_center_facility_health_servi.field_vet_center.yml]: My bad, @kevwalsh - I should have done that.

**Comment by swirtSJW** [comment]
@ethanteague Conflicted.  Needs counseling  ;)

**Review by swirtSJW — DISMISSED** [review]
Looks good.  Just waiting on tests to complete.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**